aws_lambda_builders/workflow.py

```diff
@@ -362,8 +362,10 @@ def run(self):
 
         for action in self.actions:
             action_info = "{}:{}".format(self.NAME, action.NAME)
-
-            LOG.info("Running %s", action_info)
+            function_name = ""
+            if self.options is not None and "build_logical_id" in self.options:
```

Comment:
nit: This could be simplified to `if self.options and self.options.get("build_logical_id", None):`.